rules/falco_rules.yaml

```diff
@@ -3135,6 +3135,17 @@
   priority: CRITICAL
   tags: [process, mitre_privilege_escalation]
 
+# This rule helps detect CVE-2022-4092:
+# A privilege escalation allowing container escaping
+- rule: Linux Cgroup Container Escape Vulnerability (CVE-2022-4092)
+  desc: "Detect an attempt to exploit a container escape vulnerability in the Linux Kernel."
+  condition:
+    spawned_process and container and proc.name = "unshare" and proc.args contains "mount" and proc.args contains "-o rdma" and proc.args contains "/release_agent"
```

Comment:
According to the https://sysdig.com/blog/detecting-mitigating-cve-2022-0492-sysdig/, shouldn't the `unshare` be a parent process ?